### ISSUE TITLE
Instructions not properly working on Ubuntu-20.04

### ISSUE THREAD

**Issue by tyr898** [hubisan/emacs-wsl#34]
I followed all the instructions in this guide, and tried doing the batch installation operation several times, but some packages always seem to fail installing. This is also interfering with other installations like installing zathura (I get the same errors when I install zathura now. I was also surprised when the error message mentioned things like `Setting up dictionaries-common (1.28.1) ...
Install emacsen-common for emacs
emacsen-common: Handling install of emacsen flavor emacs
/usr/lib/emacsen-common/packages/install/emacsen-common: 17: emacs: not found
ERROR: install script from emacsen-common package failed
` . Seems like I will have to go back to Ubuntu-18.04 at this rate. I just wanted to document this for future purposes. I am also using WSL2.

```bash
me:~/$ sudo apt install -y autoconf automake autotools-dev bsd-mailx build-essential \
>     diffstat gnutls-dev imagemagick libasound2-dev libc6-dev libdatrie-dev \
>     libdbus-1-dev libgconf2-dev libgif-dev libgnutls28-dev libgpm-dev libgtk2.0-dev \
>     libgtk-3-dev libice-dev libjpeg-dev liblockfile-dev liblqr-1-0 libm17n-dev \
>     libmagickwand-dev libncurses5-dev libncurses-dev libotf-dev libpng-dev \
>     librsvg2-dev libsm-dev libthai-dev libtiff5-dev libtiff-dev libtinfo-dev libtool \
>     libx11-dev libxext-dev libxi-dev libxml2-dev libxmu-dev libxmuu-dev libxpm-dev \
>     libxrandr-dev libxt-dev libxtst-dev libxv-dev quilt sharutils texinfo xaw3dg \
>     xaw3dg-dev xorg-dev xutils-dev zlib1g-dev libjansson-dev libxaw7-dev \
>     libselinux1-dev libmagick++-dev libacl1-dev gir1.2-javascriptcoregtk-4.0 \
>     gir1.2-webkit2-4.0 libenchant1c2a libglvnd-core-dev libicu-le-hb-dev \
>     libidn2-0-dev libjavascriptcoregtk-4.0-dev liboss4-salsa2 libsoup2.4-dev \
>     libsystemd-dev libwebkit2gtk-4.0-dev libx11-xcb-dev libxcb-dri2-0-dev \
>     libxcb-dri3-dev libxcb-glx0-dev libxcb-present-dev libxshmfence-dev \
>     x11proto-composite-dev x11proto-core-dev x11proto-damage-dev \
>     x11proto-fixes-dev
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'libgnutls28-dev' instead of 'gnutls-dev'
autoconf is already the newest version (2.69-11.1).
automake is already the newest version (1:1.16.1-4ubuntu6).
autotools-dev is already the newest version (20180224.1).
bsd-mailx is already the newest version (8.1.2-0.20180807cvs-1).
build-essential is already the newest version (12.8ubuntu1).
diffstat is already the newest version (1.63-1).
libacl1-dev is already the newest version (2.2.53-6).
libdatrie-dev is already the newest version (0.2.12-3).
libgif-dev is already the newest version (5.1.9-1).
libgpm-dev is already the newest version (1.20.7-5).
libice-dev is already the newest version (2:1.0.10-0ubuntu1).
libidn2-0-dev is already the newest version (2.2.0-2).
libjansson-dev is already the newest version (2.12-1build1).
libjpeg-dev is already the newest version (8c-2ubuntu8).
liblockfile-dev is already the newest version (1.16-1.1).
libm17n-dev is already the newest version (1.8.0-2).
libncurses-dev is already the newest version (6.2-0ubuntu2).
libncurses5-dev is already the newest version (6.2-0ubuntu2).
libotf-dev is already the newest version (0.9.13-7).
libpng-dev is already the newest version (1.6.37-2).
libselinux1-dev is already the newest version (3.0-1build2).
libsm-dev is already the newest version (2:1.2.3-1).
libsoup2.4-dev is already the newest version (2.70.0-1).
libthai-dev is already the newest version (0.1.28-3).
libtiff-dev is already the newest version (4.1.0+git191117-2build1).
libtiff5-dev is already the newest version (4.1.0+git191117-2build1).
libtinfo-dev is already the newest version (6.2-0ubuntu2).
libtool is already the newest version (2.4.6-14).
libxaw7-dev is already the newest version (2:1.0.13-1).
libxcb-dri2-0-dev is already the newest version (1.14-2).
libxcb-dri3-dev is already the newest version (1.14-2).
libxcb-glx0-dev is already the newest version (1.14-2).
libxcb-present-dev is already the newest version (1.14-2).
libxext-dev is already the newest version (2:1.3.4-0ubuntu1).
libxi-dev is already the newest version (2:1.7.10-0ubuntu1).
libxml2-dev is already the newest version (2.9.10+dfsg-5).
libxmu-dev is already the newest version (2:1.1.3-0ubuntu1).
libxmuu-dev is already the newest version (2:1.1.3-0ubuntu1).
libxpm-dev is already the newest version (1:3.5.12-1).
libxrandr-dev is already the newest version (2:1.5.2-0ubuntu1).
libxshmfence-dev is already the newest version (1.3-1).
libxt-dev is already the newest version (1:1.1.5-1).
libxtst-dev is already the newest version (2:1.2.3-1).
libxv-dev is already the newest version (2:1.0.11-1).
sharutils is already the newest version (1:4.15.2-4build1).
x11proto-composite-dev is already the newest version (1:2019.2-1ubuntu1).
x11proto-core-dev is already the newest version (2019.2-1ubuntu1).
x11proto-damage-dev is already the newest version (1:2019.2-1ubuntu1).
x11proto-fixes-dev is already the newest version (1:2019.2-1ubuntu1).
xorg-dev is already the newest version (1:7.7+19ubuntu14).
libenchant1c2a is already the newest version (1.6.0-11.3build1).
libgconf2-dev is already the newest version (3.2.6-6ubuntu1).
libgtk2.0-dev is already the newest version (2.24.32-4ubuntu4).
libicu-le-hb-dev is already the newest version (1.0.3+git180724-3build2).
liblqr-1-0 is already the newest version (0.4.2-2.1).
liboss4-salsa2 is already the newest version (4.2-build2010-5ubuntu5).
quilt is already the newest version (0.65-3).
texinfo is already the newest version (6.7.0.dfsg.2-5).
xaw3dg is already the newest version (1.5+E-18.3).
xaw3dg-dev is already the newest version (1.5+E-18.3).
xutils-dev is already the newest version (1:7.7+5ubuntu2).
gir1.2-javascriptcoregtk-4.0 is already the newest version (2.28.4-0ubuntu0.20.04.1).
gir1.2-webkit2-4.0 is already the newest version (2.28.4-0ubuntu0.20.04.1).
libasound2-dev is already the newest version (1.2.2-2.1ubuntu2).
libc6-dev is already the newest version (2.31-0ubuntu9.1).
libdbus-1-dev is already the newest version (1.12.16-2ubuntu2.1).
libglvnd-core-dev is already the newest version (1.3.1-1ubuntu0.20.04.1).
libgnutls28-dev is already the newest version (3.6.13-2ubuntu1.3).
libgtk-3-dev is already the newest version (3.24.20-0ubuntu1).
libjavascriptcoregtk-4.0-dev is already the newest version (2.28.4-0ubuntu0.20.04.1).
librsvg2-dev is already the newest version (2.48.7-1ubuntu0.20.04.1).
libsystemd-dev is already the newest version (245.4-4ubuntu3.2).
libwebkit2gtk-4.0-dev is already the newest version (2.28.4-0ubuntu0.20.04.1).
libx11-dev is already the newest version (2:1.6.9-2ubuntu1.1).
libx11-xcb-dev is already the newest version (2:1.6.9-2ubuntu1.1).
zlib1g-dev is already the newest version (1:1.2.11.dfsg-2ubuntu1.1).
imagemagick is already the newest version (8:6.9.10.23+dfsg-2.1ubuntu11.1).
libmagick++-dev is already the newest version (8:6.9.10.23+dfsg-2.1ubuntu11.1).
libmagickwand-dev is already the newest version (8:6.9.10.23+dfsg-2.1ubuntu11.1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
109 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Setting up dictionaries-common (1.28.1) ...
Install emacsen-common for emacs
emacsen-common: Handling install of emacsen flavor emacs
/usr/lib/emacsen-common/packages/install/emacsen-common: 17: emacs: not found
ERROR: install script from emacsen-common package failed
dpkg: error processing package dictionaries-common (--configure):
 installed dictionaries-common package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of aspell-en:
 aspell-en depends on dictionaries-common (>= 0.49.2); however:
  Package dictionaries-common is not configured yet.

dpkg: error processing package aspell-en (--configure):
 dependency problems - leaving unconfigured
No apport report written because the error message indicates its a followup error from a previous failure.
                                                                                                          dpkg: dependency problems prevent configuration of aspell:
 aspell depends on dictionaries-common (>> 0.40); however:
  Package dictionaries-common is not configured yet.

dpkg: error processing package aspell (--configure):
 dependency problems - leaving unconfigured
No apport report written because the error message indicates its a followup error from a previous failure.
                                                                                                          dpkg: dependency problems prevent configuration of hunspell-en-us:
 hunspell-en-us depends on dictionaries-common (>= 0.10); however:
  Package dictionaries-common is not configured yet.

dpkg: error processing package hunspell-en-us (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libenchant-2-2:amd64:
 libenchant-2-2:amd64 depends on hunspell-en-us | hunspell-dictionary | myspell-dictionary | aspell-dictionary | ispell-dictionary; however:
  Package hunspell-en-us is not configured yet.
  Package hunspell-dictionary is not installed.
  Package hunspell-en-us which provides hunspell-dictionary is not configured yet.
  Package myspell-dictionary is not installed.
  Package aspell-dictionary is not installed.
  Package aspell-en which provides aspell-dictionary is not configured yet.
  Package ispell-dictionary is not installed.

dpkg: error processing package libenchant-2-2:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libenchant1c2a:amd64:
 libenchant1c2a:amd64 depends on hunspell-en-us | hunspell-dictionary | aspell-dictionary | myspell-dictionary | ispell-dictionary; however:
  Package hunspell-en-us is not configured yet.
  Package hunspell-dictionary is not installed.
  Package hunspell-en-us which provides hunspell-dictionary is not configured yet.
  Package aspell-dictionary is not installed.
  Package aspell-en which provides aspell-dictionary is not configured yet.
  Package myspell-dictionary is not installed.
  Package ispell-dictionary is not installed.

dpkg: error processing package libenchant1c2a:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of enchant-2:
 enchant-2 depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.

dpkg: error processing package enchant-2 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of enchant:
 enchant depends on libenchant1c2a (>= 1.6.0); however:
  Package libenchant1c2a:amd64 is not configured yet.

dpkg: error processing package enchant (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libwebkit2gtk-4.0-37:amd64:
 libwebkit2gtk-4.0-37:amd64 depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.

dpkg: error processing package libwebkit2gtk-4.0-37:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libgoa-backend-1.0-1:amd64:
 libgoa-backend-1.0-1:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.21.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libgoa-backend-1.0-1:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libgspell-1-2:amd64:
 libgspell-1-2:amd64 depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.

dpkg: error processing package libgspell-1-2:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gir1.2-webkit2-4.0:amd64:
 gir1.2-webkit2-4.0:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.27.90); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package gir1.2-webkit2-4.0:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of shotwell:
 shotwell depends on libwebkit2gtk-4.0-37 (>= 2.15.90); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package shotwell (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gnome-control-center:
 gnome-control-center depends on libgoa-backend-1.0-1 (>= 3.10.0); however:
  Package libgoa-backend-1.0-1:amd64 is not configured yet.

dpkg: error processing package gnome-control-center (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gnome-maps:
 gnome-maps depends on gir1.2-webkit2-4.0; however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.

dpkg: error processing package gnome-maps (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libevolution:
 libevolution depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.
 libevolution depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.
 libevolution depends on libwebkit2gtk-4.0-37 (>= 2.24); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libevolution (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit:
 gedit depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package gedit (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-online-accounts:
 gnome-online-accounts depends on libgoa-backend-1.0-1 (= 3.36.0-1ubuntu1); however:
  Package libgoa-backend-1.0-1:amd64 is not configured yet.
 gnome-online-accounts depends on libwebkit2gtk-4.0-37 (>= 2.12.0); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package gnome-online-accounts (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-commander:
 gedit-plugin-commander depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            No apport report written because MaxReports is reached already
               gedit-plugin-commander depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-commander (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libedataserverui-1.2-2:amd64:
 libedataserverui-1.2-2:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.15.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libedataserverui-1.2-2:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of evolution:
 evolution depends on libevolution (>= 3.36.4); however:
  Package libevolution is not configured yet.
 evolution depends on libevolution (<< 3.37); however:
  Package libevolution is not configured yet.
 evolution depends on libwebkit2gtk-4.0-37 (>= 2.24); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error procNo apport report written because MaxReports is reached already
                                                                              No apport report written because MaxReports is reached already
                                                                                                                                            essing package evolution (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libevview3-3:amd64:
 libevview3-3:amd64 depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.

dpkg: error processing package libevview3-3:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of rhythmbox-plugins:
 rhythmbox-plugins depends on gir1.2-webkit2-4.0; however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.

dpkg: error processing package rhythmbox-plugins (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gnome-contacts:
 gnome-contacts depends on libedataserverui-1.2-2 (>= 3.16.2); however:
  Package libedataserverui-1.2-2:amd64 is not configured yet.

dpkg: error processing package gnome-contacts (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome:
 gnome depends on gnome-maps (>= 3.30); however:
  Package gnome-maps is not configured yet.
 gnome depends on shotwell | gnome-photos (>= 3.30); however:
  Package shotwell is not configured yet.
  Package gnome-photos is not installed.
 gnome depends on evolution (>= 3.30); however:
  Package evolution is not configured yet.
 gnome depends on rhythmbox-plugins; however:
  Package rhythmbox-plugins is not configured yet.

dpkg: error processing package gnome (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-multi-edit:
 gedit-plugin-multi-edit depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-multi-edit depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-multi-edit (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of evolution-plugins:
 evolution-plugins depends on libevolution (>= 3.36.4); however:
  Package libevolution is not configured yet.
 evolution-plugins depends on libevolution (<< 3.37); however:
  Package libevolution is not configured yet.
 evolution-plugins depends on libwebkit2gtk-4.0-37 (>= 2.24); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.
 evolution-plugins depends on evolution (= 3.36.4-0ubuntu1); however:
  Package evolution is not configured yet.

dpkg: error processing package evolution-plugins (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of yelp:
 yelp depends on libwebkit2gtk-4.0-37 (>= 2.19.2); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            No apport report written because MaxReports is reached already
              dpkg: error processing package yelp (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libyelp0:amd64:
 libyelp0:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.19.2); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libyelp0:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-bookmarks:
 gedit-plugin-bookmarks depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-bookmarks depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-bookmarks (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of evolution-plugin-bogofilter:
 evolution-plugin-bogofilter depends on libevolution (>= 3.36.4); however:
  Package libevolution is not configured yet.
 evolution-plugin-bogofilter depends on libevolution (<< 3.37); however:
  Package libevolution is not configured yet.
 evolution-plugin-bogofilter depends on evolution (= 3.36.4-0ubuntu1); however:
  Package evolution is not configured yet.

dpkg: error processing package evolution-plugin-bogofilter (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit-plugin-git:
 gedit-plugin-git depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-git depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-git (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of zenity:
 zenity depends on libwebkit2gtk-4.0-37 (>= 2.15.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package zenity (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-translate:
 gedit-plugin-translate depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-translate depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-translate (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of libwebkit2gtk-4.0-dev:amd64:
 libwebkit2gtk-4.0-dev:amd64 depends on libwebkit2gtk-4.0-37 (= 2.28.4-0ubuntu0.20.04.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.
 libwebkit2gtk-4.0-dev:amd64 depends on gir1.2-webkit2-4.0 (= 2.28.4-0ubuntu0.20.04.1); however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.

dpkg: error processing package libwebkit2gtk-4.0-dev:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-find-in-files:
 gedit-plugin-find-in-files depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-find-in-files depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-find-in-files (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of mutter:
 mutter depends on zenity; however:
  Package zenity is not configured yet.

dpkg: error processing package mutter (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gedit-plugin-color-picker:
 gedit-plugin-color-picker depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-color-picker depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-color-picker (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-getting-started-docs:
 gnome-getting-started-docs depends on yelp; however:
  Package yelp is not configured yet.

dpkg: error processing package gnome-getting-started-docs (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gedit-plugin-terminal:
 gedit-plugin-terminal depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-terminal depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-terminal (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gnome-software:
 gnome-software depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.

dpkg: error processing package gnome-software (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-documents:No apport report written because MaxReports is reached already

 gnome-documents depends on gir1.2-webkit2-4.0; however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.
 gnome-documents depends on libevview3-3 (>= 3.13.91); however:
  Package libevview3-3:amd64 is not configured yet.

dpkg: error processing package gnome-documents (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-bracket-completion:
 gedit-plugin-bracket-completion depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                             gedit-plugin-bracket-completion depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-bracket-completion (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-word-completion:
 gedit-plugin-word-completion depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-word-completion depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-word-completion (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of evince:
 evince depends on libevview3-3 (= 3.36.7-0ubuntu1); however:
  Package libevview3-3:amd64 is not configured yet.

dpkg: error processing package evince (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gnome-shell:
 gnome-shell depends on mutter (>= 3.36.0); however:
  Package mutter is not configured yet.

dpkg: error processing package gnome-shell (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of ubuntu-session:
 ubuntu-session depends on gnome-shell (>= 3.36.0-2ubuntu1~); however:
  Package gnome-shell is not configured yet.

dpkg: error processing package ubuntu-session (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gdm3:
 gdm3 depends on gnome-shell (>= 3.19.92); however:
  Package gnome-shell is not configured yet.

dpkg: error processing package gdm3 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-draw-spaces:
 gedit-plugin-draw-spaces depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-draw-spaces depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package gedit-plugin-draw-spaces (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-character-map:
 gedit-plugin-character-map depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-character-map depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-character-map (--configure):
 dependency problems - leaving unconfigured
dpkg: too many errors, stopping
No apport report written because MaxReports is reached already
                                                              Errors were encountered while processing:
 dictionaries-common
 aspell-en
 aspell
 hunspell-en-us
 libenchant-2-2:amd64
 libenchant1c2a:amd64
 enchant-2
 enchant
 libwebkit2gtk-4.0-37:amd64
 libgoa-backend-1.0-1:amd64
 libgspell-1-2:amd64
 gir1.2-webkit2-4.0:amd64
 shotwell
 gnome-control-center
 gnome-maps
 libevolution
 gedit
 gnome-online-accounts
 gedit-plugin-commander
 libedataserverui-1.2-2:amd64
 evolution
 libevview3-3:amd64
 rhythmbox-plugins
 gnome-contacts
 gnome
 gedit-plugin-multi-edit
 evolution-plugins
 yelp
 libyelp0:amd64
 gedit-plugin-bookmarks
 evolution-plugin-bogofilter
 gedit-plugin-git
 zenity
 gedit-plugin-translate
 libwebkit2gtk-4.0-dev:amd64
 gedit-plugin-find-in-files
 mutter
 gedit-plugin-color-picker
 gnome-getting-started-docs
 gedit-plugin-terminal
 gnome-software
 gnome-documents
 gedit-plugin-bracket-completion
 gedit-plugin-word-completion
 evince
 gnome-shell
 ubuntu-session
 gdm3
 gedit-plugin-draw-spaces
 gedit-plugin-character-map
Processing was halted because there were too many errors.
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

**Comment by hubisan** [comment]
Not using 20.04 so far. Maybe it works when using only the packages listed in the snap (snaps themselves are not working yet):
https://github.com/alexmurray/emacs-snap/blob/master/snapcraft.yaml

**Comment by hubisan** [comment]
Just installed Ubuntu 20.04 LTS and then installed Emacs with the instructions from this readme. It worked, didn't see any errors at all.
![image](https://user-images.githubusercontent.com/4160043/96352139-570af300-10c1-11eb-84e8-6aab1c3f8cdc.png)

@sailikk
Did you run `sudo apt update` and `sudo apt upgrade` beforehand?

**Comment by tyr898** [comment]
`sudo apt update` works fine. `sudo apt upgrade` gives this error:
```
me:~$ sudo apt upgrade
Reading package lists... Done
Building dependency tree
Reading state information... Done
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
109 not fully installed or removed.
After this operation, 0 B of additional disk space will be used.
Do you want to continue? [Y/n]
Setting up dictionaries-common (1.28.1) ...
Install emacsen-common for emacs
emacsen-common: Handling install of emacsen flavor emacs
/usr/lib/emacsen-common/packages/install/emacsen-common: 17: emacs: not found
ERROR: install script from emacsen-common package failed
dpkg: error processing package dictionaries-common (--configure):
 installed dictionaries-common package post-installation script subprocess returned error exit status 1
dpkg: dependency problems prevent configuration of aspell-en:
 aspell-en depends on dictionaries-common (>= 0.49.2); however:
  Package dictionaries-common is not configured yet.

dpkg: error processing package aspell-en (--configure):
 dependency problems - leaving unconfigured
No apport report written because the error message indicates its a followup error from a previous failure.
                                                                                                          dpkg: dependency problems prevent configuration of aspell:
 aspell depends on dictionaries-common (>> 0.40); however:
  Package dictionaries-common is not configured yet.

dpkg: error processing package aspell (--configure):
 dependency problems - leaving unconfigured
No apport report written because the error message indicates its a followup error from a previous failure.
                                                                                                          dpkg: dependency problems prevent configuration of hunspell-en-us:
 hunspell-en-us depends on dictionaries-common (>= 0.10); however:
  Package dictionaries-common is not configured yet.

dpkg: error processing package hunspell-en-us (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libenchant-2-2:amd64:
 libenchant-2-2:amd64 depends on hunspell-en-us | hunspell-dictionary | myspell-dictionary | aspell-dictionary | ispell-dictionary; however:
  Package hunspell-en-us is not configured yet.
  Package hunspell-dictionary is not installed.
  Package hunspell-en-us which provides hunspell-dictionary is not configured yet.
  Package myspell-dictionary is not installed.
  Package aspell-dictionary is not installed.
  Package aspell-en which provides aspell-dictionary is not configured yet.
  Package ispell-dictionary is not installed.

dpkg: error processing package libenchant-2-2:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libenchant1c2a:amd64:
 libenchant1c2a:amd64 depends on hunspell-en-us | hunspell-dictionary | aspell-dictionary | myspell-dictionary | ispell-dictionary; however:
  Package hunspell-en-us is not configured yet.
  Package hunspell-dictionary is not installed.
  Package hunspell-en-us which provides hunspell-dictionary is not configured yet.
  Package aspell-dictionary is not installed.
  Package aspell-en which provides aspell-dictionary is not configured yet.
  Package myspell-dictionary is not installed.
  Package ispell-dictionary is not installed.

dpkg: error processing package libenchant1c2a:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of enchant-2:
 enchant-2 depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package enchant-2 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of enchant:
 enchant depends on libenchant1c2a (>= 1.6.0); however:
  Package libenchant1c2a:amd64 is not configured yet.

dpkg: error processing package enchant (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libwebkit2gtk-4.0-37:amd64:
 libwebkit2gtk-4.0-37:amd64 depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.

dpkg: error processing package libwebkit2gtk-4.0-37:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libgoa-backend-1.0-1:amd64:
 libgoa-backend-1.0-1:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.21.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libgoa-backend-1.0-1:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libgspell-1-2:amd64:
 libgspell-1-2:amd64 depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.

dpkg: error processing package libgspell-1-2:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gir1.2-webkit2-4.0:amd64:
 gir1.2-webkit2-4.0:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.27.90); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package gir1.2-webkit2-4.0:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of shotwell:
 shotwell depends on libwebkit2gtk-4.0-37 (>= 2.15.90); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package shotwell (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-control-center:
 gnome-control-center depends on libgoa-backend-1.0-1 (>= 3.10.0); however:
  Package libgoa-backend-1.0-1:amd64 is not configured yet.

dpkg: error processing package gnome-control-center (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gnome-maps:
 gnome-maps depends on gir1.2-webkit2-4.0; however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.

dpkg: error processing package gnome-maps (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of libevolution:
 libevolution depends on libenchant-2-2 (>= 2.2.3); however:
  Package libenchant-2-2:amd64 is not configured yet.
 libevolution depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.
 libevolution depends on libwebkit2gtk-4.0-37 (>= 2.24); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libevolution (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit:
 gedit depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.

dpkg: error processing package gedit (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gnome-online-accounts:
 gnome-online-accounts depends on libgoa-backend-1.0-1 (= 3.36.0-1ubuntu1); however:
  Package libgoa-backend-1.0-1:amd64 is not configured yet.
 gnome-online-accounts depends on libwebkit2gtk-4.0-37 (>= 2.12.0); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package gnome-online-accounts (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-commander:
 gedit-plugin-commander depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-commander depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package gedit-plugin-commander (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libedataserverui-1.2-2:amd64:
 libedataserverui-1.2-2:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.15.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libedataserverui-1.2-2:amd64 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of evolution:
 evolution depends on libevolution (>= 3.36.4); however:
  Package libevolution is not configured yet.
 evolution depends on libevolution (<< 3.37); however:
  Package libevolution is not configured yet.
 evolution depends on libwebkit2gtk-4.0-37 (>= 2.24); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package evolution (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libevview3-3:amd64:
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                             libevview3-3:amd64 depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.

dpkg: error processing package libevview3-3:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of rhythmbox-plugins:
 rhythmbox-plugins depends on gir1.2-webkit2-4.0; however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.

dpkg: error processing package rhythmbox-plugins (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-contacts:
 gnome-contacts depends on libedataserverui-1.2-2 (>= 3.16.2); however:
  Package libedataserverui-1.2-2:amd64 is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package gnome-contacts (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome:
 gnome depends on gnome-maps (>= 3.30); however:
  Package gnome-maps is not configured yet.
 gnome depends on shotwell | gnome-photos (>= 3.30); however:
  Package shotwell is not configured yet.
  Package gnome-photos is not installed.
 gnome depends on evolution (>= 3.30); however:
  Package evolution is not configured yet.
 gnome depends on rhythmbox-plugins; however:
  Package rhythmbox-plugins is not configured yet.

dpkg: error processing package gnome (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit-plugin-multi-edit:
 gedit-plugin-multi-edit depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-multi-edit depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-multi-edit (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of evolution-plugins:
 evolution-plugins depends on libevolution (>= 3.36.4); however:
  Package libevolution is not configured yet.
 evolution-plugins depends on libevolution (<< 3.37); however:
  Package libevolution is not configured yet.
 evolution-plugins depends on libwebkit2gtk-4.0-37 (>= 2.24); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.
 evolution-plugins depends on evolution (= 3.36.4-0ubuntu1); however:
  Package evolution is not configured yet.

dpkg: error processing package evolution-plugins (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of yelp:
 yelp depends on libwebkit2gtk-4.0-37 (>= 2.19.2); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package yelp (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of libyelp0:amd64:
 libyelp0:amd64 depends on libwebkit2gtk-4.0-37 (>= 2.19.2); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package libyelp0:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-bookmarks:
 gedit-plugin-bookmarks depends on gedit (>= 3.36.1); however:
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                              Package gedit is not configured yet.
 gedit-plugin-bookmarks depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-bookmarks (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of evolution-plugin-bogofilter:
 evolution-plugin-bogofilter depends on libevolution (>= 3.36.4); however:
  Package libevolution is not configured yet.
 evolution-plugin-bogofilter depends on libevolution (<< 3.37); however:
  Package libevolution is not configured yet.
 evolution-plugin-bogofilter depends on evolution (= 3.36.4-0ubuntu1); however:
  Package evolution is not configured yet.

dpkg: error processing package evolution-plugin-bogofilter (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit-plugin-git:
 gedit-plugin-git depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-git depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-git (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of zenity:
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                             zenity depends on libwebkit2gtk-4.0-37 (>= 2.15.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.

dpkg: error processing package zenity (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-translate:
 gedit-plugin-translate depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-translate depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-translate (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of libwebkit2gtk-4.0-dev:amd64:
 libwebkit2gtk-4.0-dev:amd64 depends on libwebkit2gtk-4.0-37 (= 2.28.4-0ubuntu0.20.04.1); however:
  Package libwebkit2gtk-4.0-37:amd64 is not configured yet.
 libwebkit2gtk-4.0-dev:amd64 depends on gir1.2-webkit2-4.0 (= 2.28.4-0ubuntu0.20.04.1); however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.

dpkg: error processing package libwebkit2gtk-4.0-dev:amd64 (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-find-in-files:
 gedit-plugin-find-in-files depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-find-in-files depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-find-in-files (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration No apport report written because MaxReports is reached already
                                                                                                              No apport report written because MaxReports is reached already
                                                                                                                                                                           No apport report written because MaxReports is reached already
                                                             of mutter:
 mutter depends on zenity; however:
  Package zenity is not configured yet.

dpkg: error processing package mutter (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-color-picker:
 gedit-plugin-color-picker depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-color-picker depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-color-picker (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of gnome-getting-started-docs:
 gnome-getting-started-docs depends on yelp; however:
  Package yelp is not configured yet.

dpkg: error processing package gnome-getting-started-docs (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-terminal:
 gedit-plugin-terminal depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-terminal depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package gedit-plugin-terminal (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-software:
 gnome-software depends on libgspell-1-2 (>= 1.8.2); however:
  Package libgspell-1-2:amd64 is not configured yet.

dpkg: error processing package gnome-software (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gnome-documents:
 gnome-documents depends on gir1.2-webkit2-4.0; however:
  Package gir1.2-webkit2-4.0:amd64 is not configured yet.
 gnome-documents depends on libevview3-3 (>= 3.13.91); however:
  Package libevview3-3:amd64 is not configured yet.

dpkg: error processing package gnome-documents (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit-plugin-bracket-completion:
 gedit-plugin-bracket-completion depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-bracket-completion depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-bracket-completion (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit-plugin-word-completion:
 gedit-plugin-word-completion depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-word-completion depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-word-completion (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: dependency problems prevent configuration of evince:
 evince depends on libevview3-3 (= 3.36.7-0ubuntu1); however:
  Package libevview3-3:amd64 is not configured yet.

dpkg: error processing package evince (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gnome-shell:
 gnome-shell depends on mutter (>= 3.36.0); however:
  Package mutter is not configured yet.

dpkg: error processing package gnome-shell (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of ubuntu-session:
 ubuntu-session depends on gnome-shell (>= 3.36.0-2ubuntu1~); however:
  Package gnome-shell is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package ubuntu-session (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gdm3:
 gdm3 depends on gnome-shell (>= 3.19.92); however:
  Package gnome-shell is not configured yet.

dpkg: error processing package gdm3 (--configure):
 dependency problems - leaving unconfigured
No apport report written because MaxReports is reached already
                                                              dpkg: dependency problems prevent configuration of gedit-plugin-draw-spaces:
 gedit-plugin-draw-spaces depends on gedit (>= 3.36.1); however:
  Package gedit is not configured yet.
 gedit-plugin-draw-spaces depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

dpkg: error processing package gedit-plugin-draw-spaces (--configure):
 dependency problems - leaving unconfigured
dpkg: dependency problems prevent configuration of gedit-plugin-character-map:
 gedit-plugin-character-map depends on gedit (>= 3.36); however:
  Package gedit is not configured yet.
 gedit-plugin-character-map depends on gedit (<< 3.37); however:
  Package gedit is not configured yet.

No apport report written because MaxReports is reached already
                                                              No apport report written because MaxReports is reached already
                                                                                                                            dpkg: error processing package gedit-plugin-character-map (--configure):
 dependency problems - leaving unconfigured
dpkg: too many errors, stopping
Errors were encountered while processing:
 dictionaries-common
 aspell-en
 aspell
 hunspell-en-us
 libenchant-2-2:amd64
 libenchant1c2a:amd64
 enchant-2
 enchant
 libwebkit2gtk-4.0-37:amd64
 libgoa-backend-1.0-1:amd64
 libgspell-1-2:amd64
 gir1.2-webkit2-4.0:amd64
 shotwell
 gnome-control-center
 gnome-maps
 libevolution
 gedit
 gnome-online-accounts
 gedit-plugin-commander
 libedataserverui-1.2-2:amd64
 evolution
 libevview3-3:amd64
 rhythmbox-plugins
 gnome-contacts
 gnome
 gedit-plugin-multi-edit
 evolution-plugins
 yelp
 libyelp0:amd64
 gedit-plugin-bookmarks
 evolution-plugin-bogofilter
 gedit-plugin-git
 zenity
 gedit-plugin-translate
 libwebkit2gtk-4.0-dev:amd64
 gedit-plugin-find-in-files
 mutter
 gedit-plugin-color-picker
 gnome-getting-started-docs
 gedit-plugin-terminal
 gnome-software
 gnome-documents
 gedit-plugin-bracket-completion
 gedit-plugin-word-completion
 evince
 gnome-shell
 ubuntu-session
 gdm3
 gedit-plugin-draw-spaces
 gedit-plugin-character-map
Processing was halted because there were too many errors.
E: Sub-process /usr/bin/dpkg returned an error code (1)
```
This is what happens right now, after I have already tried running the instructions once or twice (a few days ago). I believe that I also did `sudo apt update` and `sudo apt upgrade` before I first tried using the instructions (and they worked fine), but I started getting these types of errors when I first ran the instructions.

**Comment by hubisan** [comment]
maybe `sudo apt-get install --reinstall dictionaries-common` from https://askubuntu.com/a/297191/583545

**Comment by hubisan** [comment]
or
```
sudo dpkg --purge --force-all emacsen-common
sudo apt-get upgrade -f
```
from https://bugs.launchpad.net/ubuntu/+source/emacsen-common/+bug/1361715

Can you uninstall 20.04 and reinstall it (well, if you use it for other stuff than Emacs this is not really nice)?

**Comment by tyr898** [comment]
```
sudo dpkg --purge --force-all emacsen-common
sudo apt-get upgrade -f
```
seems to be working but now `ls`:
```
me:~$ ls
diffstat    gir1.2-webkit2-4.0  libgtk-3-dev   libmagickwand-dev  libselinux1-dev  libx11-dev       libxrandr-dev  x11proto-fixes-dev
emacs-27.1  libdbus-1-dev       libidn2-0-dev  librsvg2-dev       libsystemd-dev   libxcb-dri3-dev  x11proto-composite-dev  xaw3dg-dev
```

I don't really want this stuff lying around in my home directory haha (not the behaviour I had in 18.04). I guess I could just reinstall.

**Comment by hubisan** [comment]
Closing this. Please open a new issue if this happens again or if you have found out why this is happening.
